node/node.go

```diff
@@ -110,7 +111,7 @@ func (n *Node) Run(ctx context.Context, env *environment.Env) error {
 				*eth.Block
 			}{
 				ChainID: eth.NewChainID(n.genesis.ChainID.HexBig(), ethMetrics),
-				Block:   eth.NewBlock(blockStore, ethMetrics),
+				Block:   eth.NewBlock(blockStore, new(big.Int).SetUint64(uint64(n.genesis.ChainID)), ethMetrics),
```

Comment:
It might be nice to add a `Big()` func to `ChainID` that would just return `new(big.Int).SetUint64(uint64(id))`. Then, the `ChainID.HexBig` func could be refactored to use `ChainID.Big` internally too